middleware/audit_test.go

```diff
@@ -6,21 +6,23 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/ONSdigital/dp-api-router/middleware/mock"
-	"github.com/gorilla/mux"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/ONSdigital/dp-api-router/middleware/mock"
+	"github.com/gorilla/mux"
+
 	"github.com/ONSdigital/dp-api-router/event"
 	eventmock "github.com/ONSdigital/dp-api-router/event/mock"
 	"github.com/ONSdigital/dp-api-router/middleware"
 	"github.com/ONSdigital/dp-api-router/schema"
-	kafka "github.com/ONSdigital/dp-kafka"
-	"github.com/ONSdigital/dp-kafka/kafkatest"
+
+	kafka "github.com/ONSdigital/dp-kafka/v2"
```

Comment:
NB. Not critical but the explicit package name is unnecessary as it imports as `kafka` anyway.
```suggestion
	"github.com/ONSdigital/dp-kafka/v2"
```
There are a few other imports like this in other files.